Admin-Guide.md

```diff
@@ -74,6 +74,7 @@ This will redirect you to a page where you can specify the following course prop
 
 - **Name**: The name or course code for this course. Please ensure this is correct before creating the course as this cannot be edited later.
 - **Display Name**: A longer course name or title for users to see.
+- **Maximum File Size (bytes)**: The maximum file size (in bytes) that students are allowed to upload to MarkUs.
```

Comment:
Okay so (continuing other comment thread) I think this should be reworded to refer to file uploads in general, not just what students are allowed to upload.